src/firefox/package.json

```diff
@@ -7,7 +7,6 @@
   "license": "MPL 2.0",
   "version": "0.1",
   "permissions": {
-    "private-browsing": true,
-    "unsafe-content-script": true
+    "private-browsing": true
```

Comment:
I think we need that if we want to capture links in private windows (e.g.
copy+paste) as well as to show the icon on those windows.  I'm not sure how
important those things are to us.

On Tue, Mar 24, 2015 at 12:56 PM, salomegeo <notifications@github.com>
wrote:

> In src/firefox/package.json
> <https://github.com/uProxy/uproxy/pull/1146#discussion_r27048599>:
>
> > @@ -7,7 +7,6 @@
> >    "license": "MPL 2.0",
> >    "version": "0.1",
> >    "permissions": {
> > -    "private-browsing": true,
> > -    "unsafe-content-script": true
> > +    "private-browsing": true
>
> I think we don't need this either.
>
> We used to do login flow in private window, but we don't anymore, or if we
> do we shouldn't because we're using account chooser. Can you remove it
> while you're at it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uProxy/uproxy/pull/1146/files#r27048599>.
>